modules/serverless-enabling-tls-internal-traffic.adoc

```diff
@@ -41,3 +41,16 @@ spec:
 ----
 $ oc delete pod -n knative-serving --selector app=activator
 ----
+
+. After you change `defaultCertificate` in the Ingress Controller, you need to specify the custom certificate name in the `openshift-ingress-default-certificate` field in the `KnativeServing` custom resource. For example, if the custom certificate name is `ca-ingress-cert`, add the following configuration:
++
+[source,yaml]
+----
+...
+spec:
+  config:
+    network:
+      internal-encryption: "true"
```

Comment:
The line below with `openshift-ingress-default-certificate` is correct. But this line with `internal-encryption` is out of date. The element is now deprecated and replaced with `system-internal-tls`.

This whole documentation needs to be re-worked because it was re-implemented, see the related [slack thread](https://redhat-internal.slack.com/archives/CF5ANN61F/p1722427831813569)

So, I'd by fine keeping these two lines as they are in this PR, with a note that this needs to be reworked for 1.34 anyway.

Reply:
The re-work is tracked in https://issues.redhat.com/browse/SRVKS-1243

Reply:
@mgencur Can you tell me what do we need to keep in this section, and what to leave out?
Here is the new Transport Encryption section covering SRVKS-1243:
https://github.com/openshift/openshift-docs/pull/82435
(It would be great if you could review it as well)

Reply:
Cool. I would say the whole chapter "Enabling TLS authentication for internal traffic" should be removed from this place in this PR (It's being replaced in 1.34 by the other stuff from #82435)

And, steps 3) 4) from this chapter should be moved to the new document between these two bullets:
* Specifying transport encryption features to use
...
* Restart the Controller component

It needs to be done if `system-internal-tls` was enabled and if the user changed the defaultCertificate. And it needs to happen before restarting the controller.

Reply:
I referenced the above comment from the exact place: https://github.com/openshift/openshift-docs/pull/82435/files#r1786118899

Reply:
I've done as you proposed Martin in:
https://github.com/openshift/openshift-docs/pull/82435/commits/361a5c167989bae5689c59c9dd3881d7007352aa

I will therefore close this PR.